operator/runner/runner.go

```diff
@@ -295,7 +295,7 @@ func (r *Runner) Run(ctx context.Context, id int64) error {
 		),
 		transform.WithEnviron(environ),
 		transform.WithEnviron(r.Environ),
-		transform.WithLables(
+		transform.WithLabels(
```

Comment:
This one is strange - I didn't find any other mention of the typo string `WithLables` so I am not sure how `transform.WithLables()` ever worked???

Reply:
Hmmm - the build failed, so I had to revert this change/ that typo is needed.

Reply:
https://github.com/drone/drone-yaml/blob/master/yaml/compiler/transform/labels.go#L21 there you can see the wrong function name ;)

But AFAIK this code is anyway dead.